### PR TITLE
Coretime Swaps: Time to effect clarification

### PR DESCRIPTION
**Pull request by SBalaguer**
This adds more context on when the swap comes to effect.

### CODE REVIEW

File: docs/learn/learn-guides-coretime-swap.md [hunk 1]
```diff
@@ -102,3 +102,6 @@ Steps
 - Use the manager account on your new paraID to send the
   [swap instruction](#perform-swap-on-the-relay-chain-for-an-unlocked-parachain) directly on the
   relay chain.
+
+### Time for effect
+The swap of leases is effective immediately, however the coretime chain communicates the changes of these leases to the relay chain with every new coretime cycle. Thus, the change will come into effect in the following coretime cycle.
```
Comment (profradha) [inline docs/learn/learn-guides-coretime-swap.md]: Could we specify the duration of coretime cycle here?

Comment (profradha) [reply docs/learn/learn-guides-coretime-swap.md]: What is the best way to find out when the next coretime cycle starts?

File: docs/learn/learn-guides-coretime-swap.md [hunk 2]
```diff
@@ -102,3 +102,6 @@ Steps
 - Use the manager account on your new paraID to send the
   [swap instruction](#perform-swap-on-the-relay-chain-for-an-unlocked-parachain) directly on the
   relay chain.
+
+### Time for effect
```
Comment (profradha) [inline docs/learn/learn-guides-coretime-swap.md]:
```suggestion
### Coretime Swap - Time for Effect
```

File: docs/learn/learn-guides-coretime-swap.md (hunk 1 above)
Comment (profradha) [inline docs/learn/learn-guides-coretime-swap.md]:
```suggestion
The swap of leases is effective immediately, however the Coretime chain communicates the changes of these leases to the relay chain with every new coretime cycle. Thus, the change will come into effect in the following coretime cycle. For information on when the new coretime cycle, you can view the [coretime sales](https://coretime-polkadot.subscan.io/coretime_dashboard) page.
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**